iron-location.html

```diff
@@ -135,34 +139,54 @@
           value: false
         }
       },
+
       hostAttributes: {
         hidden: true
       },
+
       observers: [
         '_updateUrl(path, query, hash)'
       ],
+
       attached: function() {
+        this.initialize();
+      },
+
+      ready: function() {
+        if (!Polymer.version || Polymer.version >= '2') {
```

Comment:
OMG THIS IS NO LONGER NECESSARY!! This was in here because of very weird behaviors in early polymer 2.0. This was a very disgusting implementation.